Dockerfile

```diff
@@ -32,7 +32,10 @@ ENV EFS_CLIENT_SOURCE=$client_source
 RUN GOOS=${TARGETOS} GOARCH=${TARGETARCH} make aws-efs-csi-driver
 
 FROM amazonlinux:2.0.20210219.0
-RUN yum install amazon-efs-utils-1.30.1-1.amzn2.noarch -y
+# Install efs-utils from github
+RUN yum -y install git rpm-build make
+RUN git clone https://github.com/aws/efs-utils
+RUN cd efs-utils && make rpm && yum -y install build/amazon-efs-utils*rpm
```

Comment:
Do we want to make this behavior configurable, e.g. from an `ARG`, like what we do for `client_source`?

Reply:
Made it configurable